packages/docs/PropTables/StatefulTablePropTable.tsx

```diff
@@ -82,6 +82,13 @@ const tableColumnsProps: Prop[] = [
     required: true,
     description: 'Header title.',
   },
+  {
+    name: 'hideHeader',
```

Comment:
Can you think of some other names that might be more verbose of what the prop does under the surface?

Reply:
Most of mine were similar to this one - `hiddenHeader`, `headerless` (similar to the parent `Table` component), `noHeader` - none of which seemed quite right. Do you have any ideas?